### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by nathanbrakken**
Dockerize the application.

Worth noting:
It might be good to try this on heroku.

### CODE REVIEW

File: docker/DockerFile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+FROM ruby:2.5.5-alpine3.10
+ENV BUILD_DEPS="curl tar wget linux-headers imagemagick fontconfig bash libstdc++ libx11 libxrender libxext libssl1.1 ca-certificates fontconfig freetype ttf-dejavu ttf-droid ttf-freefont ttf-liberation ttf-ubuntu-font-family" \
```
Comment (danjohnson3141) [inline docker/DockerFile:2]: Are there any entries in this list that are not needed for this app? Or is that even a thing you worry about in Docker?

Comment (MikeCaputo) [reply docker/DockerFile:2]: @danjohnson3141 , this list looks pretty lean to me. Probably none of these have any huge dependencies. I like to refresh myself on this Docker article once in a while: https://www.ianlewis.org/en/creating-smaller-docker-images

Comment (nathanbrakken) [reply docker/DockerFile:2]: Getting the band back together :) 🎹

Comment (mathewkeene22) [reply docker/DockerFile:2]: Whoa, a Caputo guest appearance!

Comment (MikeCaputo) [reply docker/DockerFile:2]: :D When I saw that this was on a repo that I could see, I couldn't help but take a peek.

**Pull request merged**